### PR TITLE
Some more edits for ch.7

### PR DESCRIPTION
**Pull request by schindld**
-Not sure if there's a better option for lines 1422/1424. The console returns [Object, Object].
-I found line 1483 confusing. Not sure how to word it better. Basically trying to say that all three parameters are 0-based indices, and the 3rd parameter goes up to but doesn't include that index.

### CODE REVIEW

File: chapters/ch07.asciidoc [hunk 1]
```diff
@@ -1419,9 +1419,9 @@ const things = {
   length: 2
 }
 Array.from(things)
-// <- [<div>, <div>, <div>, ...]
+// <- [Object, Object]
```
Comment (bevacqua) [inline chapters/ch07.asciidoc]: Maybe we should extract the `apple` and `onion` to variables by those names, then reference the variables in the comment, for clarity?

File: chapters/ch07.asciidoc [hunk 2]
```diff
@@ -1480,7 +1480,7 @@ const items = [1, 2, 3, ,,,,,,,]
 // <- [1, 2, 3, undefined x 7]
 ----
 
-The function call shown below takes the +items+ array and determines that it'll start "pasting" items in the sixth position. It further determines that the items to be copied will be taken starting in the second position (zero-based), until the third position (also zero-based).
+The function call shown below takes the +items+ array and determines that it'll start "pasting" items in the sixth position (zero-based). It further determines that the items to be copied will be taken starting in the second position (zero-based), until the third position (also zero-based, but excluded).
```
Comment (bevacqua) [inline chapters/ch07.asciidoc]: Let's remove the second and third mentions of `zero-based`. In the last bit we could probably get away with `third position (not inclusive)`

**Review by bevacqua — CHANGES_REQUESTED** [review]

File: chapters/ch07.asciidoc (hunk 2 above)
Comment (schindld) [reply chapters/ch07.asciidoc]: Sounds good

File: chapters/ch07.asciidoc (hunk 1 above)
Comment (schindld) [reply chapters/ch07.asciidoc]: What do you mean? `Array.from(things, thing => thing.name)` would give `['Apple', 'Onion']`, but would be a bit redundant. But I don't think that's what you mean. Can you give me an example?

Comment (bevacqua) [reply chapters/ch07.asciidoc]: I meant

```js
const apple = {
  ...
}
const onion = {
  ...
}
const groceries = {
  0: apple,
  1: onion,
  length: 2
}
Array.from(groceries)
// <- [apple, onion]
Array.from(groceries, grocery => grocery.name)
// <- ['Apple', 'Onion']
```

Comment (schindld) [reply chapters/ch07.asciidoc]: `Array.from(groceries)` still gives `[Object, Object]`.
How about  going back to the current `things` and I change line 1421 to `const [apple, onion] = Array.from(things)`, followed by `console.log(apple.name)` and `// <- "Apple"`?

Comment (bevacqua) [reply chapters/ch07.asciidoc]: It does, but we are showing that the resulting array contains the same object reference as the `groceries` object does. Using destructuring here wouldn't show that

**Comment by schindld** [comment]
No problem. I also accidentally left groceries out. It's in now.

**Comment by bevacqua** [comment]
Thanks again! :tada:

**Pull request merged**